lms/templates/credit_notifications/credit_eligibility_email.txt

```diff
@@ -9,8 +13,7 @@ ${dashboard_link}
 
 ${_(u"Course credit can help you get a jump start on your university degree, finish a degree already started, or fulfill requirements at a different academic institution.")}
 
-${_(u'To get course credit, simply go to your edX dashboard and click the "Get Credit" button. No application, transcript, or grade report is required.')}
+${_(u'To get course credit, simply go to your edX dashboard and click the "Get Credit" button. After you purchase credit you will have access to your official academic transcript from Arizona State University.')}
```

Comment:
@srpearce I have hard code credit provider name `Arizona State University` here, since user has just become eligible for credit and that user hasn't selected any credit provider (although for now we have only one credit provider `Arizona State University`) so we cannot dynamically add name here. Any suggestions?